cfme/tests/services/test_rest_services.py

```diff
@@ -2066,3 +2087,31 @@ def test_delete_cart_from_collection(self, cart):
             tags: service
         """
         delete_resources_from_collection([cart])
+
+
+@test_requirements.rest
+@pytest.mark.tier(3)
+def test_deny_service_ordering_via_api(
+    appliance, deny_service_ordering, service_templates
+):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        caseimportance: high
+        initialEstimate: 1/10h
+        setup:
+            1. In advanced settings, update `:product:`-`:allow_api_service_ordering:` to `false`
+            2. Create a dialog, catalog and catalog item.
+        testSteps:
+            1. Order the service via API.
+        expectedResults:
+            1. Service must not be ordered and response must return error.
+
+    Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1632416
+    """
```

Comment:
[please update ```Bugzilla``` like this](https://github.com/ManageIQ/integration_tests/pull/8686/files#r273422768)